submarine-client/src/main/java/org/apache/submarine/client/cli/param/runjob/package-info.java

```diff
@@ -6,13 +6,17 @@
  * to you under the Apache License, Version 2.0 (the
  * "License"); you may not use this file except in compliance
  * with the License.  You may obtain a copy of the License at
- *     http://www.apache.org/licenses/LICENSE-2.0
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
```

Comment:
Is the original license line character number exceed 80? not sure if this changed by Wanqiang's IDE.

Reply:
@tangzhankun 
Thanks for reviewing. I following the license text in http://www.apache.org/legal/src-headers.html#headers which describes 'SOURCE FILE HEADERS FOR CODE DEVELOPED AT THE ASF'.